plotting/utils.py

```diff
@@ -18,6 +18,8 @@ def get_filename(plot_type, dataset_name, extension):
 
 
 def get_mimetype(filetype: str):
+    filetype = filetype.lower()
```

Comment:
I realized people could send all-caps file types through the API and there is no conversion, which could break stuff. So I'll stick it to lower case here.